question-service/src/main/java/com/information/center/questionservice/controller/AnswerController.java

```diff
@@ -8,40 +8,37 @@
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Pageable;
 import org.springframework.stereotype.Component;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.*;
 
 @Component
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class AnswerController implements AnswerEndpoint {
 
   private final AnswerService answerService;
 
-  @Override
-  public AnswerResponse create(@RequestBody AnswerRequest answerRequest,
-      @PathVariable("questionExternalId") String questionExternalId) {
+  @PostMapping("/question/{questionExternalId}")
```

Comment:
`@PutMapping` and others are already in the interface, is no necessary to put them here to